src/XlsxWorkSheet.h

```diff
@@ -198,7 +204,7 @@ class XlsxWorkSheet {
 
       xcell->inferType(na, trimWs, wb_.stringTable(), dateFormats_);
       CellType type = xcell->type();
-      Rcpp::RObject column = cols[col];
+      cpp11::sexp column = cpp11::as_sexp(cols[col]);
```

Comment:
Do we need this call to `as_sexp()`? I would guess no? maybe just this?

```suggestion
      cpp11::sexp column(cols[col]);
```

Reply:
I ended up having to do something similar to what we did with
`std::vector<ColType> colTypes = colTypeStrings(static_cast<SEXP>(col_types));`
Because I was getting a constructor error but let me know if there is another solution!